src/common/parseDataUtil.js

```diff
@@ -23,13 +19,40 @@ const parseDataUtil = {
       to: txObject.get('to'),
       confirmations: txObject.get('confirmations'),
       memo: txObject.get('memo'),
-      status,
-      dateTime,
+      confirmedAt: txObject.get('confirmedAt'),
+      createdAt: txObject.get('createdAt'),
+      status: txObject.get('status'),
       objectId: txObject.id,
     };
     return transaction;
   },
 
+  /**
+   * processTransaction, process transaction with transaction, isSender
+   * @param {object} transaction
+   * @param {boolean} isSender
+   */
+  processTransaction(transaction, isSender) {
```

Comment:
If the method returns something, it's better to call it a `get`.

It seems to me that this method prepares view data for a transaction, so I think it's clearer to call it getTransactionViewData